front_end/panels/react_devtools/ReactDevToolsView.ts

```diff
@@ -48,18 +54,33 @@ export class ReactDevToolsViewImpl extends UI.View.SimpleView {
 
     SDK.TargetManager.TargetManager.instance().addModelListener(
       ReactDevToolsModel,
-      Events.MessageReceived,
+      ReactDevToolsModelEvents.MessageReceived,
       this.onMessage,
       this,
     );
-
     SDK.TargetManager.TargetManager.instance().addModelListener(
       ReactDevToolsModel,
-      Events.Initialized,
+      ReactDevToolsModelEvents.Initialized,
       this.initialize,
       this,
     );
+    SDK.TargetManager.TargetManager.instance().addModelListener(
+      SDK.RuntimeModel.RuntimeModel,
+      SDK.RuntimeModel.Events.ExecutionContextCreated,
```

Comment:
Remains to be seen how this behaves when we have multiple execution contexts ( = workers etc). If the event handler has significant side effects, it might be worth being more conservative here, e.g. checking the execution context's name (which will be [`main`](https://github.com/facebook/react-native/blob/3426591bbef1d6a1966c979ada326a11596eda2b/packages/react-native/ReactCommon/jsinspector-modern/tests/JsiIntegrationTest.cpp#L69 for the main VM in React Native).

Either way this is probably fine to land, but would be good to track for the future.